docs/docs/configuration/tooltip.md

```diff
@@ -316,41 +314,23 @@ The tooltip model contains parameters that can be used to render the tooltip.
     beforeBody: string[],
     // line of text that appear after the body and before the footer
     afterBody: string[],
-    bodyFont: Font,
-    _bodyAlign: string,
-    bodySpacing: number,
 
     // Title
     // lines of text that form the title
     title: string[],
-    titleFont: Font,
-    _titleAlign: string,
-    titleSpacing: number,
-    titleMarginBottom: number,
 
     // Footer
     // lines of text that form the footer
     footer: string[],
-    footerFont: Font,
-    _footerAlign: string,
-    footerSpacing: number,
-    footerMarginTop: number,
-
-    // Appearance
-    caretSize: number,
-    caretPadding: number,
-    cornerRadius: number,
-    backgroundColor: Color,
 
     // colors to render for each item in body[]. This is the color of the squares in the tooltip
     labelColors: Color[],
     labelTextColors: Color[],
 
     // 0 opacity is a hidden tooltip
     opacity: number,
-    multiKeyBackground: Color,
-    displayColors: boolean,
-    borderColor: Color,
-    borderWidth: number
+	
+	// tooltip options
```

Comment:
this uses tabs while the rest uses spaces

Reply:
@benmccann Thank you!

Reply:
this will still need to be updated

Reply:
done